### ISSUE TITLE
Multi-node config for Google Cloud Auctionmark

### ISSUE THREAD

**Issue by danmason** [xtdb/crux#3504]
Have already done some work in this regard, should be a matter of restoring it & finishing it:
- [ ] Set up a separate multi-node job
  - Runs an init container that handles the loading phase only.
  - Runs 3 containers/nodes in parallel. 
  - Maybe run Kafka on a container as well for the shared log?  
- [ ] Write up instructions for running it.
- [ ] Bump/update the cluster resources accordingly so I can run the multinode job.

**Comment by refset** [comment]
Let's add a Google docker image also when this gets done
